app/controllers/users_controller.rb

```diff
@@ -27,9 +38,49 @@ def create
     end
   end
 
+  def destroy
+    if @user.destroy
+      flash[:success] = t "user_deleted"
+      redirect_to users_url
+    else
+      flash[:danger] = t "something_wrong"
+      redirect_to users_path
```

Comment:
44 và 47 khác gì nhau e ?
cùng chung redirect to thì cho ra ngoài if else nhé

Reply:
users_url thì tương ứng với http:localhost:3000/users còn users_path thì tương ứng với : /users ạ

Reply:
vâng ạ